inputs/elements/default_csv/economic_data.csv

```diff
@@ -0,0 +1,6 @@
+,unit,economic_data
+currency,str,NOK
+discount_factor,factor,0.06
+label,str,economic_data
+project_duration,year,30
+tax,factor,0.0 
```

Comment:
exactly, it's a backup. Because the user needs to delete (or change) csv files from the input folder if she/he doesn't want to create that component, the file might easily get lost or each user would need to set up a local copy. Ideally the default_csv folder would be readable only. If you have another idea let me know.

Reply:
Thats good and makes sense.
Lets however move this "backup copy" from inputs/.../ to /tests/inputs (to be created) - then we can include the default csv directly for our pytests. This test folder should not be touched by the user anyway, so its a poper location for this.

Reply:
moved the folder and changed the according parts in the script